tools/tags/tags.go

```diff
@@ -44,6 +44,40 @@ func (and AndSet) Lines() (ls []string) {
 	return
 }
 
+// AggregateLines aggregates all tags from a set of files into one line, and remove duplicate tags.
+func (and AndSet) AggregateLines() (ls []string) {
+	var tags []string
+
+	// Aggregates all tags from a set of files, and remove duplicate tags.
+	for _, or := range and {
+		for i := 0; i < len(or); i++ {
+			dup := 0
+
+			for _, tag := range tags {
+				if tag == or[i] {
+					dup = 1
```

Comment:
nit: you can avoid enumerating all array elements:
￼       var tags []string
        tagSet = make([string]bool)
￼
￼       // Aggregates all tags from a set of files, and remove duplicate tags.
￼       for _, or := range and {
￼               for i := 0; i < len(or); i++ {
                        if _, ok = tagSet[or[i]] {
                                continue
                        }
                        tags = append(tags, or[i])
                        tagSet[or[i]] = true
￼               }
￼       }

Reply:
+1